pysgrid/tests/write_nc_test_files.py

```diff
@@ -158,6 +198,57 @@ def roms_like_sgrid(nc_filename='test_sgrid_roms_like.nc'):
         lon_v[:] = np.random.random(size=(3, 4))
         
         
+def wrf_like_sgrid(nc_filename='test_sgrid_wrf_like.nc'):
```

Comment:
Just a suggestion: drop the `_like` in these functions and netcdf files.  The are representative of those grids, not similar (like) those grids, right?